compiler/src/dmd/dsymbol.h

```diff
@@ -430,6 +430,7 @@ namespace dmd
 {
     void addMember(Dsymbol *dsym, Scope *sc, ScopeDsymbol *sds);
     Dsymbol *search(Dsymbol *d, const Loc &loc, Identifier *ident, SearchOptFlags flags = (SearchOptFlags)SearchOpt::localsOnly);
+    Dsymbols *include(Dsymbol *d, Scope *sc);
```

Comment:
you will need to add a wrapper https://github.com/dlang/dmd/blob/master/compiler/src/dmd/cxxfrontend.d within the `dmd` namespace.

Reply:
why are we adding a wrapper?
I didn't do that for newScope method

Reply:
because it is used from C++

Reply:
oh okay, thanks